test/result/function/func_json_extract.result

```diff
@@ -34,6 +34,21 @@ json_extract({"a":[1,2,3,{"a":4}]}, $**.a)
 select json_extract('{"a":[1,2,3,{"a":4}]}','$.a[*].a');
 json_extract({"a":[1,2,3,{"a":4}]}, $.a[*].a)
 4
+select json_extract('{"a":1}','$[0]');
+json_extract({"a":1}, $[0])
+{"a": 1}
+select json_extract('{"a":1}','$[0].a');
+json_extract({"a":1}, $[0].a)
+1
+select json_extract('{"a":1}','$[0].b');
+json_extract({"a":1}, $[0].b)
+null
+select json_extract('{"a":1}','$[1]');
+json_extract({"a":1}, $[1])
+null
+select json_extract(' {"af": [1, "2", {"aaf": "bb"}],"eab":"888"} ','$**.f');
+json_extract( {"af": [1, "2", {"aaf": "bb"}],"eab":"888"} , $**.f)
+null
```

Comment:
What is $**?   :)

Reply:
the path whose suffix is f，such as f, a.f, c.d.f